### ISSUE TITLE
Ensure same split train/test in SynthPipe and DataLoader

### ISSUE THREAD

**Issue by daanknoors** [daanknoors/crn-synth#19]
SynthPipe and DataLoader classes both split the data in training and testing. We use SynthPipe during generator training and DataLoader's split during evaluation. 

Confirm that both make the same split in train/test, otherwise evaluation metrics might not be accurate. Note: random-state and stratify must be same.

**Comment by daanknoors** [comment]
Seems we only pass the training data and synthetic data to reports.score_report() - not the test data. In this function the training data is then split again in training and testing - however, this test set was part of the training data when fitting the generator. Therefore, results from metrics that rely on an unseen holdout dataset will not be accurate.

We need to use the test set that was made pre-training for fair evaluation.
